test/__snapshots__/validate-options.test.js.snap.webpack4

```diff
@@ -258,23 +258,31 @@ exports[`options validate should throw an error on the "historyApiFallback" opti
 
 exports[`options validate should throw an error on the "host" option with '' value 1`] = `
 "ValidationError: Invalid options object. Dev Server has been initialized using an options object that does not match the API schema.
- - options.host should be a non-empty string.
-   -> Allows to specify a hostname to use.
-   -> Read more at https://webpack.js.org/configuration/dev-server/#devserverhost"
+ - options.host should be a non-empty string."
 `;
```

Comment:
I see the link is not showing only happening with `non-empty` error.

Reply:
Yep, it should be fixed in `schema-utils`

Reply:
I think we can merge it as is and update snapshots after fixing it in schema utils